roles/ceph-common/tasks/checks/check_ntp_atomic.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: check ntp installation on atomic
+  command: rpm -q chrony
+  register: ntp_pkg_query
+  ignore_errors: true
+  changed_when: false
+  when: is_atomic
```

Comment:
I left it in here in case anybody in the future included this file by itself. By the same logic, the `include` below is not needed because it is already checked before ntp_atomic.yml is included. I can remove it.